### DISCUSSION TITLE
Compatibility issue with DiffEditor of VSCode (Latex)

### DISCUSSION THREAD

**Discussion by e-zz** [vscode-neovim/vscode-neovim#1659] [Q&A]
VSCode supports to open diffview to see file changes in the Source Control panel (or by clicking the `open changes` button in the top right).

But for me, with vscode-neovim enabled, diff editor is always immediately redirected to the original file (or say, it automatically jumps to the buffer we selected to compare). 

Luckily the problem only occurs when I try to open a `.tex`.

I tested with Gitlens disabled or vimtex disabled, but no luck in any case.

Is anyone in a similar situation?

**Comment by xiyaowong** [comment]
Related #1594

Do you use Windows? I also encounter this issue frequently, but I haven't been able to debug where the problem lies.

**Comment by e-zz** [comment]
> Related #1594
> 
> Do you use Windows? I also encounter this issue frequently, but I haven't been able to debug where the problem lies.

Yes on Win 10.  With similar config, I also experience the issue on Ubuntu 22.04 I think. I will try to update more info here if I had this problem again.

**Reply by xiyaowong** [reply]
This bug has been fixed.
